### ISSUE TITLE
Can I test multiple different validation datasets within one epoch？

### ISSUE THREAD

**Issue by ZYX-MLer** [Deci-AI/super-gradients#1812]
I found that in the current program, a dataloader can be passed as a validation set. May I ask if it is possible to test multiple different validation datasets within one epoch without modifying the code? Thank you.

**Comment by shaydeci** [comment]
@ZYX-MLer yes, you can pass them through the test_loaders argument, if you are training with coded rather then using configuration files.
If you are using configurations, then you can pass them in the following fashion:
```
test_dataloaders:
  cifar10: cifar10_val
  cifar10_v2: cifar10_val

dataset_params:
  train_dataloader_params:
    num_workers: 0

  val_dataloader_params:
    num_workers: 0

  test_dataset_params:
    cifar10:
      root: ./data/cifar10
      train: False
      transforms:
        - Resize:
            size: 32
        - ToTensor
        - Normalize:
            mean:
              - 0.4914
              - 0.4822
              - 0.4465
            std:
              - 0.2023
              - 0.1994
              - 0.2010
      target_transform: null
      download: True

    cifar10_v2:
      root: ./data/cifar10
      train: False
      transforms:
        - Resize:
            size: 32
        - ToTensor
        - Normalize:
            mean:
              - 0.5
              - 0.5
              - 0.5
            std:
              - 0.2
              - 0.2
              - 0.2
      target_transform: null
      download: True
```
